packages/xpetra/src/Vector/Xpetra_Vector.hpp

```diff
@@ -61,10 +61,10 @@ namespace Xpetra {
     : public virtual MultiVector< Scalar, LocalOrdinal, GlobalOrdinal, Node >
   {
   public:
-    typedef Scalar scalar_type;
-    typedef LocalOrdinal local_ordinal_type;
+    typedef Scalar        scalar_type;
```

Comment:
If you're going to change the spacing, why not also change these typedefs to C++11 - style `using` statements?

Reply:
heh, ok. My PR failed anyhow... i'm updating my local build to see if I can catch these for future testing.  I'll make these modifications with the next update.